climate_emotions_map/app.py

```diff
@@ -58,11 +58,13 @@ def disable_state_select_and_party_switch_interaction(
 @callback(
     Output("drawer", "opened"),
     Input("drawer-button", "n_clicks"),
+    Input("drawer", "opened"),
```

Comment:
```suggestion
    State("drawer", "opened"),
```

I think this might be a good use case for dash's `State` instead of `Input`, which will allow you to use the `"opened"` value without having that actually trigger the callback. (See https://community.plotly.com/t/what-is-the-difference-between-input-and-state/35219 and https://dash.plotly.com/basic-callbacks#dash-app-with-state for reference)

With this, you can also get rid of the `if` statement :)